src/Timeline/TimelineFactory.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace AppBundle\Timeline;
+
+use AppBundle\Content\MediaFactory;
+use AppBundle\Entity\Timeline\Measure;
+use AppBundle\Entity\Timeline\Profile;
+use AppBundle\Entity\Timeline\Theme;
+use AppBundle\Entity\Media;
+use AppBundle\Entity\Timeline\ThemeMeasure;
+use AppBundle\Repository\MediaRepository;
+use Cocur\Slugify\Slugify;
+use League\Flysystem\Filesystem as Storage;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\File\File;
+
+class TimelineFactory
+{
+    /**
+     * @var Filesystem|null
```

Comment:
No phpdocs on private properties which are defined via constructor injection (as the constructor already has the type hints).